eo-runtime/src/test/eo/org/eolang/rust-tests.eo

```diff
@@ -206,15 +210,12 @@
     """
     use eo_env::EOEnv;
     use eo_env::eo_enum::EO;
-    use eo_env::eo_enum::EO::{EOError, EOInt};
-    pub fn foo(env: &mut EOEnv) -> EO {
-      let v1 = env.find("$.^.a") as u32;
-      let v2 = env.find("$.^.b") as u32;
-      if(env.bind(v1 , v2, "EO-att")).is_none(){
-        return EOError("bind failed".to_string());
-      } else {
-        return EOInt(0 as i64);
-      }
+    use eo_env::eo_enum::EO::{EOInt};
+    pub fn foo(env: &mut EOEnv) -> Option<EO> {
+      let v1 = env.find("$.^.a")?;
+      let v2 = env.find("$.^.b")?;
+      env.bind(v1 , v2, "EO-att")?;
```

Comment:
@levBagryansky Should there also be 4 spaces missing at the beginning of the line with the code?